presences/src/main/resources/sql/047-MA-966-function-make-alerts-configurable.sql

```diff
@@ -0,0 +1,151 @@
+-- Return true if absence event is exclude to create alert
+CREATE OR REPLACE FUNCTION presences.absence_exclude_alert(event presences.event, structureId varchar) RETURNS BOOLEAN AS
+$BODY$
+DECLARE
+    alertRulesTypeId bigint;
+    noReasonAbsenceExclude boolean;
+BEGIN
+    SELECT reason_alert_exclude_rules_type_id FROM presences.reason_alert WHERE structure_id = structureId
+                                                          AND reason_id = event.reason_id
+                                                          AND deleted_at IS NULL
```

Comment:
Si delete is not null, alors la régle n'est pas prise en compte. C'est quelle a été supprimé.